pkg/serverless/aws/logs.go

```diff
@@ -152,3 +142,18 @@ func (l *LogMessage) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+func createStringRecordForReportLog(l *LogMessage) string {
+	stringRecord := fmt.Sprintf("REPORT RequestId: %s\tDuration: %.2f ms\tBilled Duration: %d ms\tMemory Size: %d MB\tMax Memory Used: %d MB",
```

Comment:
I used the `\t` escape sequence rather than tab literals to make the correct format of the log line more clear. The old log line also had some extra spaces in it.